web/assets/js/app.js

```diff
@@ -234,9 +234,9 @@ $(document).ready(function()
     /*************************************************************************
      * Plugin to initialize datetimepicker and register change event listener
      ************************************************************************/
-    $.fn.pmiDateTimePicker = function(dateFormat = null) {
+    $.fn.pmiDateTimePicker = function(options = {}) {
```

Comment:
This is a change in behavior to allow us to pass in override options instead of being constrained to only the `format` one.